app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,19 @@
+class FoodsController < ApplicationController
+  def index
+    response = Faraday.get('https://api.nal.usda.gov/fdc/v1/foods/search') do |request|
+      request.params['api_key'] = 'nTNqChgK4a5uwh6kyqJ0B2NCCC9IovJRJT5oLrge'
+      request.params['query'] = "sweet potatoes"
+      request.params['ingredients'] = "sweet potatoes"
```

Comment:
Currently, you're hard coding these queries. The search should come in through the form that's on the welcome page. We want to use the parameter that comes in from that `:q` field in order to assign the query value.